File: leetcode/100-999/136. Single Number/solution_2.js
/**
 * @param {number[]} nums
 * @return {number}
 */

var singleNumber = function(nums) 
{   
    for (const num of nums)
    {
        if (nums.filter(n => n === num).length === 1)
            return num
    }

    return -1
};

File: leetcode/100-999/136. Single Number/solution_2.cpp
class Solution 
{
    public:
        int singleNumber(vector<int>& nums) 
        {
            for (int num : nums)
            {
                if (count(nums.begin(), nums.end(), num) == 1) 
                    return num;
            }

            return -1;    
        }
};